Makefile

```diff
@@ -85,10 +86,22 @@ deploy-ocp: manifests kustomize
 	cd config/deploy && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=${IMG}
 	$(KUSTOMIZE) build config/deploy | oc apply -f -
 
+deploy-ocp3.11: manifests-ocp311 kustomize
+	oc get project dynatrace || oc adm new-project --node-selector="" dynatrace
+	rm -f config/deploy/kustomization.yaml
+	mkdir -p config/deploy
+	cd config/deploy && $(KUSTOMIZE) create
+	cd config/deploy && $(KUSTOMIZE) edit add base ../openshift3.11
+	cd config/deploy && $(KUSTOMIZE) edit set image "quay.io/dynatrace/dynatrace-operator:snapshot"=${IMG}
+	$(KUSTOMIZE) build config/deploy | oc apply -f -
+
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=config/crd/bases
-	python3 ./hack/customize_crds.py
+	#python3 ./hack/customize_crds.py
```

Comment:
In that script we used to remove a few parts which get added to the CRD with newer versions.
I guess for the new default we won't need it anymore but most probably for OCP 3.11? If not feel free to get rid of the script as well

Reply:
I don't think it's need anymore, since we explicitly use the old crdVersion for OCP 3.11